### PR TITLE
Us 16 talk module yakup

### PR DESCRIPTION
**Pull request by yakup84**
can you reviev my project

### CODE REVIEW

**Review by sabanusta — COMMENTED** [review]
great

**Comment by sabanusta** [comment]
great

File: src/test/java/com/cydeo/step_definitions/Hooks.java
Comment (ErkanOzulu) [inline src/test/java/com/cydeo/step_definitions/Hooks.java:1]: Your driver.closed method comment out.

**Review by ErkanOzulu — COMMENTED** [review]
All works well.Good job.You should check for typos only.

**Comment by ErkanOzulu** [comment]
All works well.Good job.You should check for typos only

**Review by ErkanOzulu — APPROVED** [review]
All works well.Good job.You should check for typos only

**Review by sabanusta — COMMENTED** [review]
great job

**Pull request merged**